umh.docs.umh.app/content/en/docs/features/umh-lite.md

```diff
@@ -0,0 +1,100 @@
+---
+title: UMH Lite
+menuTitle: UMH Lite
+description: Understand the purpose and features of the UMH Lite, as well as the differences between UMH Lite and UMH Classic.
+weight: 2000
+edition: community
+aliases:
+
+---
+
+The UMH Lite is a stripped-down version of the classic United Manufacturing Hub.
+It allows you to extract data from your shop floor and send it to your own data
+infrastructure, as well as perform network monitoring.
+
+## When should I use it?
+
+If the UMH Classic seems a bit much and all you currently need is a way to get
+your data from the shop floor to your existing data infrastructure, you are in
+the right place. The UMH Lite allows you to gather data and send it to your own
+data infrastructure. It does not offer most of the features of the UMH Classic:
+you will not install Grafana or Kafka, and you will not have a Unified
+Namespace. However, you will be able to monitor the connection health of your
+network and spot errors with a single glance.
+
+## What can I do with it?
+
+### Differences between UMH Lite and Classic
+
+The UMH Lite does not offer all features of the UMH Classic. In the table below,
+you can find a comparison between both, and check if the UMH Lite fulfills your
+needs.
+
+| Feature                   | Classic | Lite |
+|---------------------------|:-------:|:----:|
+| Historian                 |    ✓    |      |
+| Analytics                 |    ✓    |      |
+| Data Visualization        |    ✓    |      |
+| UNS (Kafka & MQTT)        |    ✓    |      |
+| Alerting                  |    ✓    |      |
+| Connectivity: OPC UA      |    ✓    |  ✓   |
+| Connectivity: Node-RED    |    ✓    |      |
+| Network Monitoring        |    ✓    |  ✓   |
+| UMH Data Model v1         |    ✓    |  ✓   |
+| Tag Browser for your UNS  |    ✓    |      |
+
+### Connect devices and add protocol converters
+
+You can connect external devices like a PLC with an OPC UA server to a running
+UMH Lite instance and contextualize the data from it with a [protocol
+converter](https://umh.docs.umh.app/docs/features/connectivity/benthos-umh/).
+For contextualization, you have to use the
+[UMH Data Model v1](https://umh.docs.umh.app/docs/datamodel/messages/).
+
+### Send data to your own infrastructure
+
+All the data that your instance is gathering is sent to your own data
+infrastructure. You can configure a target MQTT broker in the instance settings
```

Comment:
change to `You can configure an external MQTT broker in the instance setting during the instance setup. You can edit it later in the instance settings` or simmilar